presto-native-execution/src/test/java/com/facebook/presto/nativeworker/PrestoNativeQueryRunnerUtils.java

```diff
@@ -48,23 +48,19 @@ public class PrestoNativeQueryRunnerUtils
     public static final String REMOTE_FUNCTION_JSON_SIGNATURES = "remote_function_server.json";
     public static final String REMOTE_FUNCTION_CATALOG_NAME = "remote";
 
+    public static final NativeWorkerParameters NATIVE_WORKER_PARAMETERS = getNativeWorkerParameters();
```

Comment:
I've generally seen public static final use only for constant strings. Your code is checking for certain default system properties. It might be better to explicitly call getNativeWorkerParameters closer to where they are used to construct the QueryRunners.

Reply:
updated